pkg/cassandra/config/config.go

```diff
@@ -142,6 +143,11 @@ func (c *Configuration) NewCluster() *gocql.ClusterConfig {
 			EnableHostVerification: c.TLS.EnableHostVerification,
 		}
 	}
+	// If tunneling connection to C*, disable cluster autodiscovery features.
+	if c.TunnelConnection {
```

Comment:
This is a bit bespoke, I'd rather make public DisableInitialHostLookup and IgnorePeerAddr or rename this variable as `DisableAutodiscovery`. Let's go with the latter.